README.md

```diff
@@ -493,6 +450,46 @@ By default, in either dynamically or statically linked builds, binaries target t
 * ```make release-static-win64``` builds binaries on 64-bit Windows portable across 64-bit Windows systems
 * ```make release-static-win32``` builds binaries on 64-bit or 32-bit Windows portable across 32-bit Windows systems
 
+## Installing Monero from a package
+
+**DISCLAIMER: These packages are not part of this repository or maintained by this project's contributors, and as such, do not go through the same review process to ensure their trustworthiness and security.**
```

Comment:
Absolutely agreed. My original PR (which accidentally got deleted) called this out. I'm open to suggestions. Some packages just use the binaries, some pull the source code and build, some pull the source code, patch it and build. In all cases, the end-user is needing to trust the package maintainer. My favored approach is to remove all reference to the packages TBH, because, even if we vet the current packages, they can change at any time without us knowing.

Reply:
How about a something simple like **"DISCLAIMER: The Monero Project Core Team does not maintain nor endorse any of the following packages. Use them at your own risk."**